lib/chat_api/messages.ex

```diff
@@ -31,6 +31,16 @@ defmodule ChatApi.Messages do
     Repo.one(query)
   end
 
+  def count_messages_sent_by_company_user_in_conversation(conversation_id) do
```

Comment:
can we add some tests for this method in `/test/chat_api/messages_test.exs`?

(I know we haven't been great about always adding tests, but I want to get more disciplined about it 😛 )